### ISSUE TITLE
[Feature] Paper trade add fees

### ISSUE THREAD

**Issue by PtrckM** [CoinAlpha/hummingbot#1722]
**Feature Suggestion**
- Add fees when Paper trade is enabled

![image](https://user-images.githubusercontent.com/54516858/81507286-e5b75f00-932e-11ea-89a0-2ddc984b50cf.png)

![image](https://user-images.githubusercontent.com/54516858/81507296-f9fb5c00-932e-11ea-9104-7faf20e5b7fb.png)

**Comment by mifeng** [comment]
Currently, Paper Ttrading markets do not apply fees in the `history` performance calculation, which is not correct. 

Instead, fees for paper trading markets should exactly the same as the fees for each real exchange. If the fees are overriden in `fee_overrides`, those should also apply to paper trading markets.

**Comment by vic-en** [comment]
In addition to the refactor in #1779, I think it'll be better to consider making the fee method(i.e `c_get_fee`) a static method. Reason for that is simply because when in paper trade mode, an instance of the connector involved is not created. And in order for the paper trade to easily make use of the fee functionality defined for each connector, it'll be better it's defined static similar to the `split_trading_pair` defined in each connector.

I'll try to give an example of how I got it to work for HuobiMarket Connector:

1. I created a static fee method in HuobiMarket connector file unlike the existing instance method
<img width="960" alt="static" src="https://user-images.githubusercontent.com/31972210/82362367-259fd580-9a04-11ea-8f5b-b58ba0885b52.PNG">

2. I modified the `c_get_fee` method in paper_trade_market.pxy to call a static method I defined in HuobiMarket connector file.
<img width="950" alt="fee in paper" src="https://user-images.githubusercontent.com/31972210/82361829-529fb880-9a03-11ea-9e6a-2a4d111fff51.PNG">

3. Edited all necessary lines in paper_trade_market.pxy to use the modified function to get fee rather than the direct `TradeFee(0)` defined
<img width="960" alt="added fee" src="https://user-images.githubusercontent.com/31972210/82362199-e5d8ee00-9a03-11ea-959c-6e8b51529a0b.PNG">

4. Compiling and running includes the fee as intended 
![fee](https://user-images.githubusercontent.com/31972210/82362448-449e6780-9a04-11ea-8718-fb57025553d2.jpg)
